ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -544,6 +545,16 @@ route_maps:
           - "< set rule 2 as string >"
 ```
 
+#### Match Lists
+
+```yaml
+match_lists:
```

Comment:
EOS supports `match-list input prefix-ipv4 <name>` and `match-list input string <name>`. 
Example:
```eos
match-list input string test
   10 match regex 10
match-list input prefix-ipv4 test2
   match prefix-ipv4 2.2.2.2/32
   match prefix-ipv4 1.1.1.1/32
```
We should make sure that the data model is close to the cli model, so it can be extended to cover everything we have in EOS. Since the syntax varies a lot between the two match-list types, we need to build seperate data models.
I suggest a few minor changes:
```yaml
match_list:
  input:
    string:
      < match_list_1 >:
        sequence_numbers:
          < sequence_id 1 >:
            match_regex: < match string >
```
Alternatively start with `match_list_input` since `input` is the only option after `match-list`

Reply:
I think a shallower dictionary is probably better, so would go with `match_list_input`.

Do you want me to add support for `prefix-ipv4` too? Should be pretty straightforward, if the parameters always look like the ones in your example.

Reply:
No it is fine if you just add what you need and know the CLI for.

Reply:
Changed to `match_list_input` in 61f6cfa40d352d108cc6174a60d9eb02229dd482.